### ISSUE TITLE
Conflicts between hyperparameters for negative sampling?

### ISSUE THREAD

**Issue by datistiquo** [RaRe-Technologies/gensim#2843]
Hi,

I wonder if there are possible interactions/conflicts when you use negative sampling with `negative>0 `and have hierarchical softmax accidentally activated` hs=1`? The docs says that only if hs=0 negative sampling will be used (negative>0). So I can hope that still if hs=1 and `negative>0 `  hopefully _**no**_ negative sampling is used?

**Comment by piskvorky** [comment]
Please follow the issue template – especially the section about questions.
